rholang/src/test/scala/coop/rchain/rholang/PeekSpec.scala

```diff
@@ -69,12 +64,9 @@ class PeekSpec extends FlatSpec with Matchers {
       .use { runtime =>
         implicit val c = runtime.cost
         for {
-          res1       <- evaluate[Task](runtime, """for(_ <<- @1) { Nil }""")
-          _          = res1.errors shouldBe empty
-          res1a      <- evaluate[Task](runtime, """@1!!("v1")""")
-          _          = res1a.errors shouldBe empty
-          res2       <- evaluate[Task](runtime, """for(_ <- @1) { @2!("v2") }""")
-          _          = res2.errors shouldBe empty
+          _          <- evaluate[Task](runtime, """for(_ <<- @1) { Nil }""")
+          _          <- evaluate[Task](runtime, """@1!!("v1")""")
+          _          <- evaluate[Task](runtime, """for(_ <- @1) { @2!("v2") }""")
           v1Data     <- runtime.space.getData(GInt(1L))
           _          = v1Data should have size 1
           resultData <- runtime.space.getData(GInt(2L))
```

Comment:
It'd make sense to extract this and the following lines into:

`_ <- checkData(GInt(2L), Seq(GString("v2")))`

Or maybe even better:

` _ <- getData(GInt(2L)) shouldBeF Seq(GString("v2")))`

I'm going to make shouldBeF add a clue where the assertion failed in my next PR

Reply:
I would have to move `shouldBeF` because now it is accessible only in `casper`. Can do that.